core/Node.js

```diff
@@ -1396,7 +1396,11 @@ Node.prototype.update = function update (time){
 
     while (queue.length) {
         item = this._components[queue.shift()];
-        if (item && item.onUpdate) item.onUpdate(time);
+        if (item && item.onUpdate) {
+            item.onUpdate(time);
+
+            if (item.onUpdated) item.onUpdated(time);
```

Comment:
Not sure if that's better at this point. Especially considering the fact that the API is now kind of inconsistent - e.g. there is no `onPreUpdate`. Just gonna remove that.